Library/Homebrew/cmd/audit.rb

```diff
@@ -182,6 +186,10 @@ def audit_file
       problem "'__END__' was found, but 'DATA' is not used"
     end
 
+    if text =~ /inreplace .* do/ && text.has_one_sub?
```

Comment:
Also, I think we should only catch `gsub`. Because `inreplace "foo", "bar", "baz"` is equivalent to using `gsub`. If you want to use `sub`, you have to use the block.